packages/arb-provider-ethers/src/lib/wallet.ts

```diff
@@ -162,6 +162,23 @@ export class ArbWallet extends ethers.Signer {
     return this.provider._wrapTransaction(tx, tx.hash)
   }
 
+  public async transferPayment(
+    originalOwner: string,
+    newOwner: string,
+    nodeHash: string,
+    messageIndex: ethers.utils.BigNumberish
+  ): Promise<ethers.providers.TransactionResponse> {
+    const msgIndex = ethers.utils.bigNumberify(messageIndex)
+    const globalInbox = await this.globalInboxConn()
+    const tx = await globalInbox.transferPayment(
+      originalOwner,
+      newOwner,
+      nodeHash,
+      msgIndex
+    )
+    return this.provider._wrapTransaction(tx, tx.hash)
```

Comment:
This is just a standard L1 transaction so it doesn't need to be resolved by the arbitrum provider. We can just return `tx` directly instead of wrapping it. It might be better to not have this in the wallet at all and instead just put it in the token bridge sdk.